src/tr1/game/ui_new/ui_new_console.c

```diff
@@ -0,0 +1,168 @@
+typedef struct {
```

Comment:
Is this module intended to be included currently?

Reply:
Oh no ! It's a garbage from my local play.